tests/test_datasets.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+from django.urls import reverse
+from django_webtest import DjangoTestApp
+
+from vitrina.datasets.factories import DatasetFactory
+
+
+@pytest.mark.django_db
+def test_dataset_detail_view(app: DjangoTestApp):
+    dataset = DatasetFactory()
+    resp = app.get(reverse('dataset-detail', args=[dataset.slug]))
+
+    # without tags
+    assert len(resp.context['tags']) == 0
+    assert resp.context['tags'] == ""
+
+    # with tags
+    dataset.tags = "tag-1, tag-2, tag-3"
+    dataset.save()
+    resp = app.get(reverse('dataset-detail', args=[dataset.slug]))
+    assert resp.status == '200 OK'
+    assert len(resp.context['tags']) == 3
```

Comment:
Test bellow checks both, length and content, so no need to check length separately here.